src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -150,5 +152,15 @@ public GeneratedTagHelperContext()
         /// Contains information about in-scope tag helpers, HTML attributes, and the tag helpers' output.
         /// </remarks>
         public string ExecutionContextTypeName { get; set; }
+
+        /// <summary>
+        /// The name of the property in HtmlHelper used to encode html.
+        /// </summary>
+        public string HtmlEncoderPropertyName { get; set; }
+
+        /// <summary>
+        /// The name of the injected property of type IHtmlHelper.
```

Comment:
if this remains don't mention "injected" -- that's up to MVC to decide.  suggest `The name of the property containing <see cref="HtmlEncoderPropertyName"/>.`  (hmm, mentioning `IHtmlHelper` with `HtmlEncoderPropertyName` isn't worthwhile)